<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <!-- planners -->
    <rosparam file="$(find ourplanner)/config/config.yaml" command="load"/>
    <node name="global_planner_node" pkg="ourplanner" type="global_planner.py" output="screen"> </node>
    <node name="middle_planner_node" pkg="ourplanner" type="middle_planner.py" output="screen"> </node>
    <node name="local_planner_node" pkg="ourplanner" type="local_planner.py"  output="screen"> </node>  

    <!-- rviz -->
    <node name="planner_rviz" pkg="rviz" type="rviz" args="-d $(find ourplanner)/rviz/planner.rviz"/>

</launch>
